### DISCUSSION TITLE
saf-training/courses/beginner/06

### DISCUSSION THREAD

**Discussion by giscus** [mitre/saf-training#2] [Ideas]
# saf-training/courses/beginner/06

The MITRE Security Automation Framework Team's Introduction to InSpec profile development

https://mitre.github.io/saf-training/courses/beginner/06.html

**Comment by brett-w** [comment]
When I run 
`inspec exec /root/my_nginx --reporter cli json:/tmp/output.json`, 
results are printed to screen but files is not generated. 

When I remove the slash before 'tmp'
`inspec exec /root/my_nginx --reporter cli json:tmp/output.json`
output to both locations works as expected.

**Comment by karikarshivani** [comment]
Was this last block supposed to be in section # 7?
https://mitre.github.io/saf-training/courses/beginner/06.html#running-the-baseline-straight-from-github

**Comment by karikarshivani** [comment]
Assign input('nginx_modules') to a variable before the describe block with the for loop

**Comment by aaronlippold** [comment]
Open an issue with the inspec project for the use of the array input in the describe block for looping. It was working on the previous release.
